src/tailwind.css

```diff
@@ -1,8 +1,186 @@
 /* purgecss start ignore */
+
 @tailwind base;
 @tailwind components;
 /* purgecss end ignore */
 
 /* Override utils here*/
 
 @tailwind utilities;
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: #1d2530;
+}
+
+.joinUsWrapper {
+  display: flex;
+  flex-direction: column;
+}
+
+.joinUsWrapper__registerLoginWrapper {
+  display: flex;
+}
+
+.joinUsWrapper__registerLoginWrapper__register {
+  @apply -mx-5;
+  background: linear-gradient(270deg, #ffb421 11.7%, #ff7521 90.81%);
+  border-radius: 9px;
+  border-top-left-radius: 0px;
+  border-bottom-left-radius: 0px;
+  clip-path: polygon(20% 0, 100% 0, 100% 100%, 0% 100%);
+}
+
+.joinUsWrapper__registerLoginWrapper__register > h3 {
+  width: 72px;
+  height: 20px;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 21px;
+  color: #ffffff;
+  text-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.joinUsWrapper__registerLoginWrapper__register:hover,
+.joinUsWrapper__registerLoginWrapper__login:hover {
+  transition: 1s;
+  transform: scale(1.2);
+}
+
+.joinUsWrapper__registerLoginWrapper__login {
+  background: linear-gradient(270deg, #ff7521 11.7%, #ffb421 90.81%);
+  border-radius: 9px;
+  border-top-right-radius: 0px;
+  border-bottom-right-radius: 0px;
+  clip-path: polygon(0 0, 100% 0, 80% 100%, 0% 100%);
+}
+
+.joinUsWrapper__registerLoginWrapper__login > h3 {
+  width: 72px;
+  height: 20px;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 21px;
+  color: #ffffff;
+  text-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+}
+
```

Comment:
all the code above can be substituted with tail wind. if you make a decision to use tailwind, then you need to stick to the decision.

learning tailwind is really simple, the only problem you might face is like missed units ( like 20px instead or 18 px) if it isn't of utmost importance, then go with tailwind for up to 2 pixels. And extend when required.

Delete all these classes and apply the styles using tailwind, please. let all your team know too


the main issue here is that we are duplicating everything and then debugging styling can become really hard.